patches/0006-Vendor-OpenSSL-crypto-library.patch

```diff
@@ -0,0 +1,3113 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Quim Muntal <qmuntaldiaz@microsoft.com>
+Date: Mon, 23 May 2022 12:59:36 +0000
+Subject: [PATCH] Vendor OpenSSL crypto library
+
+---
```

Comment:
Should the command from the old summary get copied over here?

https://github.com/microsoft/go/blob/669b2bdadd7939392ae988cc095a2250359f2c6c/patches/0102-Vendor-OpenSSL-crypto-library.patch#L4-L7

More generally about these summaries: I don't think we need to be concerned about including big descriptions for each patch in the initial PR. But, I think we should probably plan to include more in the future, especially as we start to collaborate with other Go distro maintainers. It seems to me like the natural place for people to look to start understanding what we've done.